config/ospd-openvas.service

```
@@ -1,8 +1,8 @@
 [Unit]
 Description=OSPd Wrapper for the OpenVAS Scanner (ospd-openvas)
 Documentation=man:ospd-openvas(8) man:openvas(8)
-After=network.target networking.service redis-server@openvas.service
-Wants=redis-server@openvas.service
+After=network.target networking.service redis-server@openvas.service mosquitto.service
+Wants=redis-server@openvas.service mosquitto.service
```

```suggestion
Wants=redis-server@openvas.service mosquitto.service notus-scanner.service
```

- As a default setup of `ospd-openvas` expects the `notus-scanner` integration it probably could make sense to include this as well (`notus-scanner.service` is also pulling in `mosquitto.service`)
- Not adding it to `After` as AFACT the service could be started at any time (while `mosquitto.service` needs to be actually started before `ospd-openvas.service`